src/main/java/core/basesyntax/service/FileServiceImpl.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax.service;
+
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+
+public class FileServiceImpl implements FileService {
+    @Override
+    public List<String> read(String fileName) {
+        try {
+            return Files.readAllLines(Path.of(fileName));
+        } catch (IOException e) {
+            throw new RuntimeException("Can't read file: " + fileName);
+        }
+    }
+
+    @Override
+    public void writeToFile(String content, String toFileName) {
+        try (BufferedWriter bufferedWriter = new BufferedWriter(new FileWriter(toFileName))) {
+            bufferedWriter.write(content);
+        } catch (IOException e) {
+            throw new RuntimeException("Can't write to file: " + toFileName);
```

Comment:
When catching IOException, it's better to include the original exception when throwing the new RuntimeException to provide a complete stack trace for debugging. Change the catch block to include the original exception.